core/src/main/java/com/valtech/aem/saas/core/indexing/PathTransformerPipelineService.java

```diff
@@ -0,0 +1,40 @@
+package com.valtech.aem.saas.core.indexing;
+
+import com.valtech.aem.saas.api.indexing.PathTransformer;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+import org.osgi.service.component.annotations.ReferenceCardinality;
+import org.osgi.service.component.annotations.ReferencePolicyOption;
+
+@Slf4j
+@Component(name = "Search as a Service - Path Transformer Pipeline",
+    service = PathTransformerPipeline.class)
+public class PathTransformerPipelineService implements PathTransformerPipeline {
+
+  @Reference(cardinality = ReferenceCardinality.MULTIPLE, policyOption = ReferencePolicyOption.GREEDY)
+  private List<PathTransformer> pathTransformers = new CopyOnWriteArrayList<>();
```

Comment:
do we have a default pathTransformer? we need to have 1 which uses DayCQExternalizer